core/src/main/java/io/questdb/Bootstrap.java

```diff
@@ -188,14 +188,20 @@ public CairoConfiguration getCairoConfiguration() {
             log.errorW().$(e).$();
             throw new BootstrapException(e);
         }
-
-        // metrics
         if (config.getMetricsConfiguration().isEnabled()) {
             metrics = Metrics.enabled();
         } else {
             metrics = Metrics.disabled();
             log.advisoryW().$("Metrics are disabled, health check endpoint will not consider unhandled errors").$();
         }
+        Unsafe.setWriterMemLimit(config.getCairoConfiguration().getWriterMemoryLimit());
+        long rssMemoryLimit = config.getCairoConfiguration().getRssMemoryLimit();
+        if (rssMemoryLimit > 0) {
+            log.advisoryW().$(String.format("Setting RSS memory limit to %,d bytes", rssMemoryLimit)).$();
```

Comment:
nit:
```suggestion
            log.advisoryW().$("Setting RSS memory limit to ").$(rssMemoryLimit).$("bytes").$();
```

Reply:
I'm already changing this code, but why not prefer a more readable format "123,456,789 bytes" over "123456789 bytes"?